src/components/Comments/Comment.js

```diff
@@ -160,7 +164,7 @@ const Comment = ({
   const onPressDMAnon = async () => {
     try {
       setLoading({...loading, loadingDmAnon: true});
-      await sendMessageDM(comment?.id, 'comment', 'ANONYMOUS');
+      await sendMessageDM(comment?.id, 'comment', 'ANONYMOUS', feedId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendMessageDM` function now includes an additional parameter `feedId`. Ensure that all calls to this function have been updated to include this new argument.



<!-- This is an auto-generated comment by OSS CodeRabbit -->